### PR TITLE
Grap 51

### PR DESCRIPTION
**Pull request by svyatoslavkk**
Add Query editor and JSON viewer, and demo functionality

### CODE REVIEW

File: src/components/queryEditor/styles.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
```
Comment (ppmarkek) [inline src/components/queryEditor/styles.ts:1]: Use mui styled and mui components

File: src/pages/ErrorPage/styles.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import styled, { keyframes } from 'styled-components';
+
```
Comment (ppmarkek) [inline src/pages/ErrorPage/styles.ts]: Use mui styled and mui components

**Review by ppmarkek — APPROVED** [review]

File: src/pages/ErrorPage/ErrorPage.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <Container>
+      <div className="background-img">
+        <Space></Space>
+        <Wrapper>
+          <div className="img-wrapper">
+            <Digit>44</Digit>
+          </div>
+          <Text>
+            The page you are trying to search has been moved to another
```
Comment (1LisaW) [inline src/pages/ErrorPage/ErrorPage.tsx]: Could you please add here i18n? Text should be dependand on current language.

File: src/pages/ErrorPage/ErrorPage.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            universe.
+          </Text>
+          <Button onClick={handleButtonClick}>GET ME HOME</Button>
```
Comment (1LisaW) [inline src/pages/ErrorPage/ErrorPage.tsx]: Could you please add here i18n? Text should be dependand on current language.

**Review by 1LisaW — APPROVED** [review]

**Pull request merged**